### ISSUE TITLE
Relative requires aren't registered/identified

### ISSUE THREAD

**Issue by Reapimus** [NightrainsRbx/RobloxLsp#267]
When requiring a module relative to the current script, the LSP is unable to find the module. It can find modules perfectly fine from absolute paths but just fails for relative paths for whatever reason.

I've checked and made sure there were no issues with my rojo project setup, it also happens with both a single lua file and a folder with an init.lua file.

**Comment by akolar1** [comment]
It seems like you may benefit from checking out the Roblox platform to possibly find solutions or resources related to your module importing issue. Please visit https://www.roblox.com/ for more information.

**Comment by Reapimus** [comment]
> It seems like you may benefit from checking out the Roblox platform to possibly find solutions or resources related to your module importing issue. Please visit https://www.roblox.com/ for more information.

This won't solve my issue. It's a problem **with the LSP** as I have clearly stated I am using Rojo in my project. Neither do I connect the LSP to any place file through a plugin like it supports since I am most commonly not writing code with studio open at all times.

**Comment by maybeSomeDev** [comment]
Yeah I am facing the same issue right now. I though it was something on my end or in the settings because I didn't find anyone talking about it in the DevForum.

To further explain the problem:

The script I'm in: `StarterGui.PluginUI.main`
![Image](https://github.com/user-attachments/assets/964d9ac3-2515-42b8-99fd-a1abc8f7b816)

- The problem: 
> When requiring a module relative to the current script, the LSP is unable to find the module

![Image](https://github.com/user-attachments/assets/d261a220-056f-403c-b307-81c5019322f2)

- The temporary fix:

![Image](https://github.com/user-attachments/assets/2f979ddc-bf85-411a-858c-ff5245ce4cdf)

But if you tried to use `PluginUI:FindFirstChild` in a new line it won't show, and will only show using `.`
